types/common.ts

```diff
@@ -46,15 +46,35 @@ export interface ClaimDetailsResult {
   monetaryStatus: string
 }
 
+export interface NullClaimDetailsResult {
+  benefitYearStartDate: null
+  benefitYearEndDate: null
+  claimBalance: null
+  weeklyBenefitAmount: null
+  lastPaymentIssued: null
+  lastPaymentAmount: null
+}
+
 export interface Claim {
   ClaimType?: null | undefined | string
-  uniqueNumber?: null | string
-  claimDetails?: null | ClaimDetailsResult
-  hasPendingWeeks?: null | undefined | boolean // deprecated for hasValidPendingWeeks
-  hasValidPendingWeeks?: null | undefined | boolean
-  hasCertificationWeeksAvailable?: null | undefined | boolean
-  isBYE?: null | undefined | boolean
-  pendingDetermination?: null | PendingDetermination[]
+  uniqueNumber: null | string
+  claimDetails: null | ClaimDetailsResult
+  hasPendingWeeks: boolean // deprecated for hasValidPendingWeeks
+  hasValidPendingWeeks: boolean
+  hasCertificationWeeksAvailable: boolean
+  isBYE: boolean
+  pendingDetermination: null | PendingDetermination[]
+}
+
+export interface NullClaim {
```

Comment:
By pulling `NullClaim` & `NullClaimDetailsResult` out into their own types, we can now narrow on them in `queryApiGateway` - TS then can tell when we return from `queryApiGateway` that we _must_ have `Claim`  instead of `NullClaim` - allowing us to assert not only that we expect certain fields exist, but their value in non-`null`, as well.